MapboxNavigation/RouteMapViewController.swift

```diff
@@ -557,11 +508,13 @@ extension RouteMapViewController: NavigationViewDelegate {
 
     func navigationMapViewDidStartTrackingCourse(_ mapView: NavigationMapView) {
         self.navigationView.resumeButton.isHidden = true
+        self.navigationView.resumeButton.alpha = 0
```

Comment:
adjusting alpha is needed to make animations looks good, here it would get out of sync, so need to adjust it as well